docs/UserGuide.md

```diff
@@ -1,12 +1,5 @@
----
-  layout: default.md
-  title: "User Guide"
-  pageNav: 3
----
+# WedLog User Guide
 
-# AB-3 User Guide
-
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a  Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
```

Comment:
Similarly, I think you could copy our basic description of WedLog from `docs/index.md` over here too!